### ISSUE TITLE
[Bug]: Page fails to render in <webview> and crashes application on next navigation

### ISSUE THREAD

**Issue by Catlinz** [electron/electron#28477]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

11.1.1

### What operating system are you using?

Windows

### Operating System Version

Windows 10 version 1909

### What arch are you using?

x64

### Last Known Working Electron version

10.0.0-beta,2

### Expected Behavior

A `<webview>` should be able to load a Page (in this case, instagram) and display it.  Application should not crash when trying to navigated away from the page.

### Actual Behavior

If a `<webview>` tries to load some pages (e.g., Instagram) the `<webview>` doesn't render the page and then if you try and navigate the `<webview>` away from the page, the entire Application crashes without any visible error messages.

I'm not sure if this is occurring on other websites as well, but https://www.instagram.com was the one that I randomly stumbled upon that was causing this behaviour.

The issue seems to have been introduced in 10.0.0-beta.3, since all versions before that seem to be working, including latest version of 9.x.x.

### Testcase Gist URL

https://gist.github.com/6c9b8407a897163b4ae8a8078b75d3ce

**Comment by nornagon** [comment]
Stack trace:

```
Crash reason:  EXC_BAD_ACCESS / KERN_INVALID_ADDRESS
Crash address: 0x50
Process uptime: 4 seconds

Thread 0 (crashed)
 0  Electron Framework!content::RenderFrameProxyHost::SetChildRWHView(content::RenderWidgetHostView*, gfx::Size const*) [memory : 2602 + 0x0]
    rax = 0x0000000000000000   rdx = 0x0000000000000000
    rcx = 0x0000000000000005   rbx = 0x0000000000000000
    rsi = 0x00007fb7ea0e7a00   rdi = 0x0000000000000000
    rbp = 0x00007ffeebd4ebd0   rsp = 0x00007ffeebd4ebc0
     r8 = 0x0000000000000002    r9 = 0x0000000000000002
    r10 = 0x0000000000000002   r11 = 0x0000000000000000
    r12 = 0x00007fb7eba70100   r13 = 0x00007fb7e6892800
    r14 = 0x0000000000000000   r15 = 0x00007fb7e788be00
    rip = 0x000000010a0e7f7d
    Found by: given as instruction pointer in context
 1  Electron Framework!content::WebContentsImpl::ReattachToOuterWebContentsFrame() [web_contents_impl.cc : 2306 + 0xb]
    rbp = 0x00007ffeebd4ec50   rsp = 0x00007ffeebd4ebe0
    rip = 0x000000010cd70942
    Found by: previous frame's frame pointer
 2  Electron Framework!content::WebContentsImpl::NotifyViewSwapped(content::RenderViewHost*, content::RenderViewHost*) [web_contents_impl.cc : 6289 + 0x9]
    rbp = 0x00007ffeebd4ed50   rsp = 0x00007ffeebd4ec60
    rip = 0x000000010a18c911
    Found by: previous frame's frame pointer
 3  Electron Framework!content::WebContentsImpl::NotifySwappedFromRenderManager(content::RenderFrameHost*, content::RenderFrameHost*, bool) [web_contents_impl.cc : 7755 + 0xe]
    rbp = 0x00007ffeebd4ee00   rsp = 0x00007ffeebd4ed60
    rip = 0x000000010a198ed1
    Found by: previous frame's frame pointer
 4  Electron Framework!content::RenderFrameHostManager::CommitPending(std::__1::unique_ptr<content::RenderFrameHostImpl, std::__1::default_delete<content::RenderFrameHostImpl> >, std::__1::unique_ptr<content::BackForwardCacheImpl::Entry, std::__1::default_delete<content::BackForwardCacheImpl::Entry> >, bool) [render_frame_host_manager.cc : 3159 + 0xd]
    rbp = 0x00007ffeebd4ef10   rsp = 0x00007ffeebd4ee10
    rip = 0x000000010a0de182
    Found by: previous frame's frame pointer
 5  Electron Framework!content::RenderFrameHostManager::CommitPendingIfNecessary(content::RenderFrameHostImpl*, bool, bool, bool) [render_frame_host_manager.cc : 380 + 0x12]
    rbp = 0x00007ffeebd4ef50   rsp = 0x00007ffeebd4ef20
    rip = 0x000000010a0ddeb6
    Found by: previous frame's frame pointer
 6  Electron Framework!content::RenderFrameHostManager::DidNavigateFrame(content::RenderFrameHostImpl*, bool, bool, bool, blink::FramePolicy const&) [render_frame_host_manager.cc : 349 + 0x5]
    rbp = 0x00007ffeebd4ef80   rsp = 0x00007ffeebd4ef60
    rip = 0x000000010a0ddd88
    Found by: previous frame's frame pointer
 7  Electron Framework!content::Navigator::DidNavigate(content::RenderFrameHostImpl*, content::mojom::DidCommitProvisionalLoadParams const&, std::__1::unique_ptr<content::NavigationRequest, std::__1::default_delete<content::NavigationRequest> >, bool) [navigator.cc : 382 + 0x13]
    rbp = 0x00007ffeebd4f200   rsp = 0x00007ffeebd4ef90
    rip = 0x000000010a0b8f85
    Found by: previous frame's frame pointer
 8  Electron Framework!content::RenderFrameHostImpl::DidCommitNavigationInternal(std::__1::unique_ptr<content::NavigationRequest, std::__1::default_delete<content::NavigationRequest> >, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadParams>, mojo::InlinedStructPtr<content::mojom::DidCommitSameDocumentNavigationParams>) [render_frame_host_impl.cc : 8854 + 0xb]
    rbp = 0x00007ffeebd4f340   rsp = 0x00007ffeebd4f210
    rip = 0x000000010a0c6991
    Found by: previous frame's frame pointer
 9  Electron Framework!content::RenderFrameHostImpl::DidCommitNavigation(content::NavigationRequest*, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadParams>, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadInterfaceParams>) [render_frame_host_impl.cc : 9317 + 0x11]
    rbp = 0x00007ffeebd4f470   rsp = 0x00007ffeebd4f350
    rip = 0x000000010a0c5f68
    Found by: previous frame's frame pointer
10  Electron Framework!base::internal::Invoker<base::internal::BindState<void (content::RenderFrameHostImpl::*)(content::NavigationRequest*, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadParams>, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadInterfaceParams>), base::internal::UnretainedWrapper<content::RenderFrameHostImpl>, content::NavigationRequest*>, void (mojo::StructPtr<content::mojom::DidCommitProvisionalLoadParams>, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadInterfaceParams>)>::RunOnce(base::internal::BindStateBase*, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadParams>&&, mojo::StructPtr<content::mojom::DidCommitProvisionalLoadInterfaceParams>&&) [bind_internal.h : 498 + 0xb]
    rbp = 0x00007ffeebd4f4b0   rsp = 0x00007ffeebd4f480
    rip = 0x000000010a0dc97d
    Found by: previous frame's frame pointer
11  Electron Framework!content::mojom::NavigationClient_CommitNavigation_ForwardToCallback::Accept(mojo::Message*) [callback.h : 101 + 0x6]
    rbp = 0x00007ffeebd4f560   rsp = 0x00007ffeebd4f4c0
    rip = 0x00000001096bda9d
    Found by: previous frame's frame pointer
[...]
```

[528d637e-95d6-476a-ada8-a334bddbb994.dmp.zip](https://github.com/electron/electron/files/6245753/528d637e-95d6-476a-ada8-a334bddbb994.dmp.zip)

Bisected to https://github.com/electron/electron/compare/v10.0.0-beta.2...v10.0.0-beta.3. Unfortunately nothing obvious stands out there other than the Chromium roll, and even in there nothing that we changed seems like a likely culprit. So it's probably due to some internal Chromium change that happened [in that range](https://chromium.googlesource.com/chromium/src/+log/17da1064b910f4170320ff37e971064916aed1dc..85.0.4161.2?n=10000).

**Comment by Kilian** [comment]
seems related to https://github.com/electron/electron/issues/25469 as well?

**Comment by nornagon** [comment]
@Kilian hm, maybe, but that bug is present in 9.x while this one appeared in v10.0.0-beta.3.

**Comment by alectrocute** [comment]
+1

**Comment by Catlinz** [comment]
Just confirming that this is still an issue on the latest Electron versions (14.0.0 and 15.0.0-beta.1)

**Comment by Catlinz** [comment]
Just updating that this is still an issue in all versions of electron up to the latest alpha (17.0.0-alpha.5) unfortunately.

**Comment by Catlinz** [comment]
A happy update!  So it looks like this issue (at least using my gist) appears to have been fixed between the v18.0.0-alpha.5 and v18.0.0-beta.1 Electron versions!

**Comment by Catlinz** [comment]
Going to close this since it's fixed in Electron v18.0.0 and up.
